rules/bridges.yaml

```diff
@@ -39,13 +40,8 @@ CABHUT:
 
 ^LowBridge:
 	Inherits: ^LowBridgeRamp
```

Comment:
I believe that this ^LowBridge is now totally invulnerable due removal of Targetable: as no parent has targetable